include/nbl/video/utilities/CScanner.h

```diff
@@ -143,270 +142,43 @@ prefix sum with subgroup sized workgroups at peak Bandwidth efficiency in about
 
```

Comment:
update the comments about the algo between lines 112-141 because we can't spinwait, we need to outline the new algo properly